automation/run-tests.sh

```diff
@@ -34,8 +34,12 @@ test -t 1 && USE_TTY="-t"
 function remove_container {
     res=$?
     [ "$res" -ne 0 ] && echo "*** ERROR: $res"
-    container_exec 'rm -rf $CONTAINER_WORKSPACE/*nmstate*.rpm'
-    ${CONTAINER_CMD} rm $CONTAINER_ID -f
+    if [ "$CONTAINER_CMD" == "docker" ];then
```

Comment:
`if $CONTAINER_CMD --version | grep -qv podman; then` should run this only if it is not podman. With podman-docker installed, the docker command will actually be podman but show this in the version.

Reply:
Done, thanks.